node/process_test.ts

```diff
@@ -326,6 +326,66 @@ Deno.test({
   },
 });
 
+Deno.test({
+  name: "process.stdin readable with a TTY",
+  ignore: !Deno.isatty(Deno.stdin.rid),
```

Comment:
Leave a TODO comment like: `TODO: Run this test even in non tty environment`

Reply:
A TODO comment has been added. (c10b492)